adafruit_emc2101/emc2101_lut.py

```diff
@@ -366,7 +366,7 @@ def external_temp_high_limit(self):
         full_tmp >>= 5
         full_tmp *= 0.125
         if not -64 <= full_tmp <= 127:
-            raise EMC2101BadValueException()
+            raise RuntimeError("Connection")
```

Comment:
Same thing here about using `OSError` if catching a faulty communication with the sensor